lms/templates/discussion/_discussion_inline.html

```diff
@@ -12,3 +12,15 @@
         <button class="new-post-btn btn-neutral btn-small">${_("Add a Post")}</button>
     % endif
 </div>
+<script type="text/javascript">
+/* global DiscussionModuleView */
```

Comment:
I believe that changes to the java/coffeescript code also affect the tabs, so we were supposed to leave them unchanged for this round.  @e-kolpakov  can you confirm?